### ISSUE TITLE
Transpile cache.dir option doesn't seem to work

### ISSUE THREAD

**Issue by sethhall** [radarsu/ts-import#20]
Is this supposed to be a supported feature?  The project I'm working on uses [Yarn 2 with PnP](https://yarnpkg.com/features/pnp) so there is no `node_modules/` directory which causes me to get the following error:

```
Error: EROFS: read-only filesystem, mkdir '/node_modules/ts-import/cache'
```

I tried setting the cache directory to a different location but it doesn't seem to take the setting.  I found the setting here:

https://github.com/radarsu/ts-import/blob/7a1590ade48e733ccdae1cccd789d2a5231ae099/src/load.interfaces.ts#L19-L23

**Comment by radarsu** [comment]
There was a bug regarding using this parameter, That should be fixed in `4.0.0-beta.8` version.

**Comment by sethhall** [comment]
Thanks!
